install-space builds error linking against  libsns_ik_util.so

As I was setting up the CI jobs for ROS Indigo, I seem to have run into a bug (which is the whole point of setting up the CI in the first place :) The command:
```
/bin/sh -c catkin config --extend /opt/ros/$ROS_DISTRO --install --cmake-args -DCMAKE_BUILD_TYPE=Release &&     catkin build --catkin-make-args run_tests
```
The error:
```
Starting >>> sns_ik_examples                                                   
_______________________________________________________________________________
Errors << sns_ik_examples:install /root/ws_sns_ik/logs/sns_ik_examples/build.install.000.log
/usr/bin/ld: warning: libsns_ik_util.so, needed by /root/ws_sns_ik/install/lib/libsns_ik.so, not found (try using -rpath or -rpath-link)
/root/ws_sns_ik/install/lib/libsns_ik.so: undefined reference to `sns_ik::pinv_QR(Eigen::Matrix<double, -1, -1, 0, -1, -1> const&, Eigen::Matrix<double, -1, -1, 0, -1, -1>*, double)'
/root/ws_sns_ik/install/lib/libsns_ik.so: undefined reference to `sns_ik::pinv_damped_P(Eigen::Matrix<double, -1, -1, 0, -1, -1> const&, Eigen::Matrix<double, -1, -1, 0, -1, -1>*, Eigen::Matrix<double, -1, -1, 0, -1, -1>*, double, double)'
/root/ws_sns_ik/install/lib/libsns_ik.so: undefined reference to `sns_ik::pinv_forBarP(Eigen::Matrix<double, -1, -1, 0, -1, -1> const&, Eigen::Matrix<double, -1, -1, 0, -1, -1> const&, Eigen::Matrix<double, -1, -1, 0, -1, -1>*)'
/root/ws_sns_ik/install/lib/libsns_ik.so: undefined reference to `sns_ik::isIdentity(Eigen::Matrix<double, -1, -1, 0, -1, -1> const&)'
/root/ws_sns_ik/install/lib/libsns_ik.so: undefined reference to `sns_ik::pinv_QR_Z(Eigen::Matrix<double, -1, -1, 0, -1, -1> const&, Eigen::Matrix<double, -1, -1, 0, -1, -1> const&, Eigen::Matrix<double, -1, -1, 0, -1, -1>*, Eigen::Matrix<double, -1, -1, 0, -1, -1>*, double, double)'
/root/ws_sns_ik/install/lib/libsns_ik.so: undefined reference to `sns_ik::pinv(Eigen::Matrix<double, -1, -1, 0, -1, -1> const&, Eigen::Matrix<double, -1, -1, 0, -1, -1>*, double)'
collect2: error: ld returned 1 exit status
make[2]: *** [/root/ws_sns_ik/devel/.private/sns_ik_examples/lib/sns_ik_examples/all_ik_tests] Error 1
make[1]: *** [CMakeFiles/all_ik_tests.dir/all] Error 2
make: *** [all] Error 2
cd /root/ws_sns_ik/build/sns_ik_examples; catkin build --get-env sns_ik_examples | catkin env -si  /usr/bin/make install; cd -
...............................................................................
Failed << sns_ik_examples:install               [ Exited with code 2 ] 
```

--- a/sns_ik_lib/CMakeLists.txt
+++ b/sns_ik_lib/CMakeLists.txt
@@ -33,11 +33,6 @@ catkin_package(
 include_directories(include utilities ${catkin_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS})
 link_directories(${orocos_kdl_LIBRARY_DIRS})
 
-# library for private utilities
-add_library(sns_ik_util
-            utilities/sns_ik_math_utils.cpp
-            utilities/sns_linear_solver.cpp)
-
 # library for public API
 add_library(sns_ik
             src/fosns_velocity_ik.cpp
@@ -46,8 +41,10 @@ add_library(sns_ik
             src/osns_velocity_ik.cpp
             src/sns_ik.cpp
             src/sns_position_ik.cpp
-            src/sns_velocity_ik.cpp)
-target_link_libraries(sns_ik sns_ik_util ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
+            src/sns_velocity_ik.cpp
+            utilities/sns_ik_math_utils.cpp
+            utilities/sns_linear_solver.cpp)
+target_link_libraries(sns_ik ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
 
 # install the public API
 install(TARGETS sns_ik LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})

